src/notifications/screens/notifications.screen.js

```diff
@@ -187,6 +187,7 @@ class Notifications extends Component {
     markAsReadByDispatch(notification.id);
     navigation.navigate('Issue', {
       issueURL: notification.subject.url.replace('pulls', 'issues'),
+      isPR: notification.subject.type === 'PullRequest',
```

Comment:
@housseindjirdeh Hmm.. it's good to do, although I did not plan to do it in this PR, that's why it's called that.

I noticed that in events PR is not clickable (there is no handler `onPress`). It will also navigate to issue, although they will not have an `issue` key, then you need to pass the `pull_request` key. There will be something like this:

```js
this.props.navigation.navigate('Issue', {
  issue: userEvent.payload.issue || userEvent.payload.pull_request,
  isPR: typeof userEvent.payload.pull_request !== undefined,
});
```

This is ok?

And I get crash when I want to go to issue, although I have not done this before (or probably not on all issues):
![image](https://user-images.githubusercontent.com/4408379/28553564-1e14e4e6-70fd-11e7-881a-b9a61e31bd19.png)